lua/weapons/e.r.c.s.lua

```diff
@@ -0,0 +1,96 @@
+SWEP.Author = "el_patito_loco"
+SWEP.PrintName = "e.r.c.s" --"Experimental Rotating Chamber Sniper"
+SWEP.Category = TASWeapons.Category
+SWEP.Author = TASWeapons.Authors.El_Patito
+SWEP.Instructions = [[Left Click shoots a powerful 12.7 mm sniper round that will destroy your enemies!!
+At the moment our engineers are working on new features which will provide more mass destruction!!
+]]
+
+SWEP.Spawnable = true
+SWEP.AdminOnly = false
+
+SWEP.ViewModel = "models/weapons/v_357.mdl"
+SWEP.WorldModel = "models/weapons/w_357.mdl"
+
+local ShootSound = Sound("weapons/mortar/mortar_fire1.wav")
+
+SWEP.FiresUnderwater = true
+SWEP.DrawWeaponInfoBox = true
+SWEP.IconOverride = "materials/entities/weapon_357.png"
+
+--ammo
+
+SWEP.Primary.ClipSize = -1
+SWEP.Primary.DefaultClip = -1
+SWEP.Primary.Automatic = true
+SWEP.Primary.Ammo = "none"
+
+SWEP.Secondary.ClipSize = -1
+SWEP.Secondary.DefaultClip = -1
+SWEP.Secondary.Automatic = true
+SWEP.Secondary.Ammo = "none"
+
+--Inventory related
+
+SWEP.Weight = 5
+SWEP.AutoSwitchTo = false
+SWEP.AutoSwitchFrom = false
+
+SWEP.Slot = 4
+SWEP.SlotPos = 3
+SWEP.DrawAmmo = false
+SWEP.DrawCrosshair = true
+
+--weapon properties
+
+SWEP.Primary.Recoil = 1
+SWEP.Primary.Damage = 75
+SWEP.Primary.NumShots = 10
+SWEP.Primary.Spread = 0
+SWEP.Primary.Delay = 1
+SWEP.MouseWasDown = true
+
+function SWEP:Initialize()
+	self:SetHoldType("revolver")
+end
+
+function SWEP:PrimaryAttack()
+	local ply = self:GetOwner()
+
+	local Bullet = {}
```

Comment:
nit: There's inconsistent variable naming throughout this file. Could you stick to `camelCase` throughout?

Reply:
Named it owner (I guess you only refer to ply variable)

Reply:
I was referring to `Bullet` and `ShootSound`